sources/packages/web/src/components/partial-view/student/NavBar.vue

```diff
@@ -30,18 +30,20 @@
 <script lang="ts">
 import { computed } from "vue";
 import { AppConfigService } from "../../../services/AppConfigService";
+import { routeConstants } from "../../../constants/routes/RouteConstants";
 export default {
   setup() {
     const menuItems: any = [];
     const isAuthenticated = computed(
       () => AppConfigService.shared.authService?.authenticated === true,
     );
+    const studentProfileEditPath = routeConstants.STUDENTPROFILEEDIT;
```

Comment:
I believe that there is a way to expose the constants globally to us be able to access them directly on the template without the need to declare and expose it through the setup every time.
It is just an alternative way.
The route names in the constants is already a better more organized way 👍